package/yast2-network.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed Oct  2 09:05:14 UTC 2019 - Josef Reidinger <jreidinger@suse.com>
+
+- Fix again network CLI with improved error reporting (bsc#1151242)
```

Comment:
"again" in the changelog looks like .. not ugly but something that happen with some .. let say frequency :P

Reply:
agree, I want to mention that network-ng. So I will reformulate.